staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -119,9 +120,11 @@ type Config struct {
 	// FlowControl, if not nil, gives priority and fairness to request handling
 	FlowControl utilflowcontrol.Interface
 
-	EnableIndex     bool
-	EnableProfiling bool
-	EnableDiscovery bool
+	EnableIndex       bool
+	EnableProfiling   bool
+	EnableDiscovery   bool
+	EnableDiscoveryV1 bool
```

Comment:
I dont like how the new discovery is V1 and the old discovery is unversioned. Suggestions are appreciated. Would feel slightly weird to introduce new endpoint /discovery/v2 for the first version. Suggestions appreciated